ui/scripts/enos-test-ember.js

```diff
@@ -45,9 +45,8 @@ const testHelper = require('./test-helper');
   try {
     const testArgs = ['test', '-c', 'testem.enos.js'];
 
-    if (process.env.TEST_FILTERS) {
-      const filters = JSON.parse(process.env.TEST_FILTERS).map((filter) => '-f=' + filter);
-      testArgs.push(...filters);
+    if (process.env.TEST_FILTER) {
+      testArgs.push('-f=' + process.env.TEST_FILTER);
```

Comment:
nice improvement

Reply:
yeah, I learned that it's not possible to add more than one filter.